src/js/controllers/walletHome.js

```diff
@@ -1297,7 +1297,7 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
 
         confirmDialog.show(msg, function(confirmed) {
           if (confirmed) {
-            self._doSendMax(resp.amount * self.satToUnit);
+            self._doSendMax(parseFloat((resp.amount * self.satToUnit).toFixed(8)));
```

Comment:
This is not right. Take a look at https://github.com/bitpay/copay/blob/master/src/js/controllers/walletHome.js#L611
The nb of decimal places is dependent on the selected unit.